tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
+    "module": "es2015",
```

Comment:
Why is this necessary?

Reply:
It errors if you don't. There was a change to `rollup-plugin-typescript2` which adds the error.